HW0/P5/P5.py

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+import matplotlib.pyplot as plt
+bag = np.array(list(xrange(300))) + 1
+t_single = bag - 1
+t_infinite = np.ceil([np.log2(i) for i in bag])
+plt.plot(bag, t_single, '-k', label='Single Cashier')
+plt.plot(bag, t_infinite, '-b', label='Infinite Cashiers')
+plt.legend()
+plt.show()
```

Comment:
Plot looks good. I think it might be more clear what's going on if you used a log scale on the y axis. Labeled axes are also helpful!